orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/tasks/v2/CreateV2PipelineTemplateTask.java

```diff
@@ -66,15 +67,27 @@ public TaskResult execute(Stage stage) {
 
     validate(pipelineTemplate);
 
-    Response response = front50Service.saveV2PipelineTemplate((Map<String, Object>) stage.decodeBase64(
-      "/pipelineTemplate",
-      Map.class,
-      pipelineTemplateObjectMapper
-    ));
+    Response response;
+    String version = (String) stage.getContext().get("version");
+    if (StringUtils.isEmpty(version)) {
```

Comment:
I don't think you need this `if`, can't you just call saveV2PipelineTemplate with an empty version?

Reply:
Yep that was a relic from the first try as a path parameter.